services/iot/client.go

```diff
@@ -46,6 +46,9 @@ type IoTClient interface {
 	IngestNodeDataStream(c <-chan api.IngestNodeDataStreamInput) error
 	IngestNodeDataStreamWithContext(ctx context.Context, c <-chan api.IngestNodeDataStreamInput) error
 
+	GetLatestNodeData(input *api.GetLatestNodeDataInput) (api.NodeData, error)
+	GetLatestNodeDataContext(ctx context.Context, input *api.GetLatestNodeDataInput) (api.NodeData, error)
```

Comment:
moves away from the convention `WithContext`?

Reply:
as well as pointers in the input

Reply:
Its more carelessness, i will fix it